Cargo.toml

```diff
@@ -80,8 +80,8 @@ members = [
 
 [workspace.dependencies]
 alloy-consensus = { git = "https://github.com/alloy-rs/alloy.git", rev = "d68a6b787b2904061f0ae7fcc02ece8513e3c500"}
```

Comment:
Can we update this as well?

Reply:
I will just remove it as we just use a constant from it so it is a little overkill

Reply:
nit: You could have used `alloy_consensus: v0.3.1`, as we already depend on it indirectly. Up to you.